include/alpaka/dev/DevUniformCudaHipRt.hpp

```diff
@@ -65,8 +65,16 @@ namespace alpaka
         {
             return !((*this) == rhs);
         }
+        int iDevice() const
+        {
+            return m_iDevice;
+        }
+        int setDevice(int devIdx)
```

Comment:
I would prefer if we would hold to common practice and have the methods be called `getXxx` and `setXxx` where `Xxx` might be `DeviceIndex`, `DeviceIdx` or `Device`.

Reply:
We can, but in this case, according to the issue opened by Andrea, I would also rename also the iDevice() method for OpenAcc and OpenMp5 to have the same "interface".